packages/api/helper/googleDrive.js

```diff
@@ -0,0 +1,49 @@
+const { google } = require('googleapis');
+const parentFolderId = process.env.APPLICATION_FOLDER_ID
+const clientEmail  = process.env.SERVICE_CLIENT_EMAIL
+const privateKey = process.env.SERVICE_PRIVATE_KEY
```

Comment:
Can you look at the way that the Google Sheets private key is being parsed and apply that logic here too? It'd be nice to make that into a helper function and use that everywhere we read in a private key.

Reply:
Okay, I think I have figured out a couple of implementations for this. Which is the better of the two for this case?

I can either create a parsePrivateKey() function and pass a pk, and return it parsed. Or have privateKeys() and call on it to retrieve the parsed values like privateKeys.gDrive or privateKeys.gSheets, a getter essentially.

Reply:
Addressed in latest commit